### DISCUSSION TITLE
I have a problem to install security onion last version standalone

### DISCUSSION THREAD

**Discussion by codefaztaz** [Security-Onion-Solutions/securityonion#11445] [2.4]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16 gb

### Storage for /

500 gb

### Storage for /nsm

100 gb

### Network Traffic Collection

tap

### Network Traffic Speeds

more than 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I am installing security onion in a pc with the minimum requirements with standalone version. I set up the two nic cards with the wan and lan ips. and when the installer is installing the packages. there is an error but I can not find the logs path. I dont know if I did a mistake with the wan and lan ips. does securityonion must have internet to install all packets? or?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by Acewiza** [comment]
Swapping the monitor/admin ports might shed light if you think the LAN/WAN interfaces are backwards.  But you're probably wasting ur time trying to monitor 10gig of traffic on a minimum install.

**Comment by codefaztaz** [comment]
I was able to install security onion but with eval mode not standalone. I setup wan interface 192.168.1.117 and lan interface with 192.168.0.85 but after the successful instalation , securityonion says to access the web administration on 192.168.1.117 but I set up on   192.168.0.85 I don't underestand what is going

**Comment by dougburks** [comment]
In most cases, folks that install in eval mode set a static IP address on their management interface and use the second interface for sniffing network traffic (no IP address at all). 

For more information, please see:
https://docs.securityonion.net/en/2.4/hardware.html#nic
https://docs.securityonion.net/en/2.4/architecture.html#evaluation
